lib/src/main/java/com/basistheory/android/model/ElementValueReference.kt

```diff
@@ -10,6 +11,8 @@ class ElementValueReference(private val valueGetter: () -> String?) {
         _element = element
     }
 
+    override fun toString(): String = registerExpressionValue(getValue())
```

Comment:
One worry here is that calling toString multiple times returns different ids, but maybe that's not a problem 🤷